app/views/users/sessions/new.html.erb

```diff
@@ -36,7 +36,8 @@
         </div>
         <% if devise_mapping.rememberable? -%>
           <div class="form-group newproject-checkbox">
-            <label class="primary-checkpoint-container users-primary-checkpoint"><h6>Remember Me</h6>
+           <h6>Remember Me</h6>
+            <label class="primary-checkpoint-container users-primary-checkpoint">
```

Comment:
This change isn't required. We want the checkbox to be accessible via the Label text.
Rest all LGTM

Reply:
It was mentioned in the issue that checkbox should be accessed only through checkbox so according to mentions in issue 
I excluded h1
if  we allow then only change we need is display:inline-block

Reply:
Will confirm this and then we can move ahead

Reply:
> Will confirm this and then we can move ahead

Any update?

Reply:
Having it accessible from the label text is a good option. Let us go with that @pavanjoshi914 . The fix you mentioned would do

Reply:
Ohk implementing following changes